repository/project.go

```diff
@@ -0,0 +1,154 @@
+package repository
+
+import (
+	"context"
+
+	"release-manager/pkg/dberrors"
+	"release-manager/repository/model"
+	"release-manager/repository/util"
+	svcmodel "release-manager/service/model"
+
+	"github.com/google/uuid"
+	"github.com/nedpals/supabase-go"
+)
+
+type ProjectRepository struct {
+	client *supabase.Client
+	entity string
+}
+
+func NewProjectRepository(c *supabase.Client) *ProjectRepository {
+	return &ProjectRepository{
+		client: c,
+		entity: "projects",
+	}
+}
+
+func (r *ProjectRepository) Create(ctx context.Context, p svcmodel.Project) (svcmodel.Project, error) {
```

Comment:
Is there any point in returning the same structure as passed as a parameter? To me it looks like this is an unnecessary overhead and simply returning an `error` would be enough.